app/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
**TODO FOR LOST POINTS**
You need ``H1-H6's`` in your index.html so you can actually use your ``scss for loop`` on them.